### ISSUE TITLE
As a data user I would like to have a new attribute for approved uses 

### ISSUE THREAD

**Issue by AndyDaniel1** [dzhw/metadatamanagement#3214]
We need to implement a new attribute on the data package detail page that describes the approved uses.  

en: "Approved purposes of use"
de: "Erlaubter Nutzungszweck"  

We should develop a controlled vocabulary for the different purposes.  (for now prio 2)

The approved purposes should also be transferred to the DLP to give the data service complete information for the review of the submitted research projects.

**Comment by AndyDaniel1** [comment]
> > We should develop a controlled vocabulary for the different purposes.
> 
>> Do we limit options by providing a list that can be chosen from or do we provide a text field with an autocomplete that >>provides previously entered purposes? @AndyDaniel1

for the first implementation I would go with an autocomplete 

>>The approved purposes should also be transferred to the DLP to give the data service complete information for the review of the submitted research projects.
>> This should be implemented similar to #3233 ("Remarks for the User Service").

yes

**Comment by AndyDaniel1** [comment]
As I elaborated a little bit more on the implementation of #3233 just a little specification here: This attribute should be visible to all data users but it should also be transferred to the DLP.

**Comment by AndyDaniel1** [comment]
@tilovillwock The publisher should be able to filter data packages according to this attribute.

**Comment by tilovillwock** [comment]
> The approved purposes should also be transferred to the DLP to give the data service complete information for the review of the submitted research projects.

@UteH @AndyDaniel1 do you have a specific place in DLP in mind where the approved purpose should be displayed?

**Comment by UteH** [comment]
discussed with @AndyDaniel1
placement: 
![grafik](https://github.com/dzhw/metadatamanagement/assets/18573122/83123cb2-7526-4ac9-ba14-6cd987586fd1)

order
- "remarks for the user service" #3233 
- "approved uses", #3214
- "annotations"

**Comment by ewolf47** [comment]
> @tilovillwock The publisher should be able to filter data packages according to this attribute.

@AndyDaniel1 We are talking about this filter (see screenshot), not the one in the sidebar. Is this correct?
![image](https://github.com/dzhw/metadatamanagement/assets/118182853/8f40ae00-2580-40db-8a0b-b3395bf5e0a2)

**Comment by AndyDaniel1** [comment]
@ewolf47 yes this is correct

**Comment by AndyDaniel1** [comment]
needs further refinement

**Comment by ewolf47** [comment]
![image](https://github.com/dzhw/metadatamanagement/assets/118182853/b94e4211-2416-4e69-9335-04ef2b4339ca)

@AndyDaniel1  Just a quick question about filtering by the approved uses in the multiple choice list (controlled vocabulary): I have implemented it according to an **"or" logic**. This means that in the example above, all data packages would be displayed that have an approved usage of either "Wissenschaftliche Nutzung" or "Kommerzielle Nutzung" or both types. In other words, if a data package has at least one of the ticked approved usages, it will be displayed.

The alternative would be an **"and"-logic**, in which only those data packages are displayed that have _all_ the approved usages that has been ticked by the user.

Intuitively, as a user, I would expect the "or"-logic. That is why I have currently implemented it in this way. However, I wanted to ask again whether this is what you want.

**Comment by AndyDaniel1** [comment]
@ewolf47 Yes, I would also expect the "or" logic as the main scenario. An "and" logic would also make sense in some cases. Could both cases be implemented with a chooser that allows you to choose between the two filter options?

**Comment by ewolf47** [comment]
In general, that would be possible. I can adapt the logic so that both ways are possible. In the end, it may just be a question of clarity/understandability in the application. But I'll try to implement it :smile:

**Comment by AndyDaniel1** [comment]
Thank you! You are right that might be a little confusing / complex, but I would give it a try in the test phase.

**Comment by ewolf47** [comment]
@AndyDaniel1 
I implemented the new filter with the optional "and-logic"-switch. I have added a relatively long tooltip, which can of course still be changed according to your wishes. It looks like this at the moment:
![image](https://github.com/dzhw/metadatamanagement/assets/118182853/3029bda1-6351-4786-b2d6-8b7fb10ab99a)

And the english version:
![image](https://github.com/dzhw/metadatamanagement/assets/118182853/0c595253-8fb3-4290-be0b-783f0bcaca13)

**Comment by AndyDaniel1** [comment]
Due the harmonization with the DLP we have to change the string "non-commercial use"

de: "nicht-kommerzielle Zwecke" --> "statistische nicht-kommerzielle Zwecke"
en: "non-commercial use" --> "statistical non-commerical use"

**Comment by tilovillwock** [comment]
@AndyDaniel1 judging from the mentions here in the ticket I assume the only thing left to do here is adjusting the strings mentioned in  this [comment](https://github.com/dzhw/metadatamanagement/issues/3214#issuecomment-2202397679)? Because the feature itself has been deployed to production already.
